ToDoApp/ViewController.swift

```diff
@@ -75,7 +75,7 @@ class ViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        self.view.backgroundColor = .systemGray6
+        self.view.backgroundColor = UIColor(red: 0.87, green: 0.86, blue: 0.82, alpha: 1.00)
         
         view.addSubview(emailTextField)
         view.addSubview(loginLabel)
```

Comment:
```suggestion
      [
        emailTextField,
        loginLabel,
        passwordTextField,
        loginButton,
        registerButton,
        forgotPasswordButton,
        forgotPasswordImageView
      ].forEach {
        view.addSubview($0)
      }
```

이런 식으로 `.forEach` 구문을 이용하면 중복된 코드를 지울 수 있어서 편리할거야!